doc/pages/Manual.md

```diff
@@ -412,6 +412,7 @@ three scopes:
    the [`with-test`](#pkgvar-with-test) and [`with-doc`](#pkgvar-with-doc)
    variables, available in the `depends:`, `depopts:`, `build:`, `install:` and
    `remove:` fields.
+   // TODO add here, do we keep it for packages filter or all ?
```

Comment:
what do you mean by that?

Reply:
do we want to keep that variable only for `depends:` & co, or we allow it also for command fields like `install`.
It is meant to be a developer setup variable... setup is just packages or commands too ?